From 84b22f50f6dcd543242e2e47b46bd08932f00ad1 Mon Sep 17 00:00:00 2001
From: Alexander Alemayhu <alexander@alemayhu.com>
Date: Tue, 18 Oct 2016 11:10:22 +0200
Subject: [PATCH 1/3] staging: rtl8712: remove redundant assignment

looks like 'pdev_network->IELength = pdev_network->IELength' and
'pdev_network->Ssid.SsidLength = pdev_network->Ssid.SsidLength' do
nothing.

Discovered using coccinelle and changed with following script:

@@
expression x;
@@

- x = x;

Signed-off-by: Alexander Alemayhu <alexander@alemayhu.com>
---
 drivers/staging/rtl8712/rtl871x_cmd.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/staging/rtl8712/rtl871x_cmd.c b/drivers/staging/rtl8712/rtl871x_cmd.c
index b7ee5e63af33..20f3f07e2616 100644
--- a/drivers/staging/rtl8712/rtl871x_cmd.c
+++ b/drivers/staging/rtl8712/rtl871x_cmd.c
@@ -437,8 +437,6 @@ u8 r8712_createbss_cmd(struct _adapter *padapter)
 	pcmd->rspsz = 0;
 	/* notes: translate IELength & Length after assign to cmdsz; */
 	pdev_network->Length = pcmd->cmdsz;
-	pdev_network->IELength = pdev_network->IELength;
-	pdev_network->Ssid.SsidLength =	pdev_network->Ssid.SsidLength;
 	r8712_enqueue_cmd(pcmdpriv, pcmd);
 	return _SUCCESS;
 }
-- 
2.10.1

